### PR TITLE
feat: add Bootstrap 5

### PR DESCRIPTION
**Pull request by Nemo-Git-Hub**
# Add Bootstrap 5
[CN-28](https://trello.com/c/HE0WSMD7)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,12 +29,17 @@
     "imagemin-webpack": "^5.1.1",
     "mini-css-extract-plugin": "^2.6.1",
     "node-sass": "^7.0.3",
+    "postcss-loader": "^7.0.1",
+    "sass": "^1.55.0",
     "sass-loader": "^13.1.0",
+    "style-loader": "^3.3.1",
```
Comment (asd-ez) [inline package.json]: Do you use these loaders?

Comment (Nemo-Git-Hub) [reply package.json]: no. It`s delete?

Comment (asd-ez) [reply package.json]: Yea

Comment (Nemo-Git-Hub) [reply package.json]: ok. +

File: package.json [hunk 2]
```diff
@@ -28,12 +29,16 @@
     "imagemin-webpack": "^5.1.1",
     "mini-css-extract-plugin": "^2.6.1",
     "node-sass": "^7.0.3",
+    "postcss-loader": "^7.0.1",
```
Comment (asd-ez) [inline package.json]: Do you use this loader?

**Review by asd-ez — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (Nemo-Git-Hub) [reply package.json]: It`s complete.

**Pull request merged**